plugins/module_utils/ndi.py

```diff
@@ -0,0 +1,464 @@
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2021, Lionel Hercot (@lhercot) <lhercot@cisco.com>
+# Copyright: (c) 2022, Cindy Zhao (@cizhao) <cizhao@cisco.com>
+# Simplified BSD License (see licenses/simplified_bsd.txt or https://opensource.org/licenses/BSD-2-Clause)
+
+from __future__ import (absolute_import, division, print_function)
+from html import entities
```

Comment:
not used think can be removed? or is it needed?

Reply:
removed